__tests__/app.test.js

```diff
@@ -106,5 +106,54 @@ describe("app", () => {
           });
       });
     });
+    describe("/articles", () => {
+      describe("GET /articles/:article_Id", () => {
+        test("200: returns an object with key 'article' containing an object", () => {
+          return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+              expect(typeof body).toBe("object");
+              expect(Array.isArray(body)).toBe(false);
```

Comment:
This line isn't necessary as the next like implicitly shows this isn't an array when we access a property on it